doc/api/https.markdown

```diff
@@ -243,3 +243,4 @@ var req = https.request(options, (res) => {
 [`tls.connect()`]: tls.html#tls_tls_connect_options_callback
 [`tls.createServer()`]: tls.html#tls_tls_createserver_options_secureconnectionlistener
 [`url.parse()`]: url.html#url_url_parse_urlstr_parsequerystring_slashesdenotehost
+[`Buffer`]: buffer.html#buffer_buffer
```

Comment:
This page maintains this list in ascending order (in fact all the pages should do this). Can you please put this in the right place?